plugin.go

```diff
@@ -156,7 +156,13 @@ func (p *Plugin) createInvalidateJob() {
 
 func (p *Plugin) runJobs() {
 	client := p.client
-	jobChan := make(chan struct{}, maxConcurrent)
+	maxConcurrency, err := strconv.Atoi(p.MaxConcurrency)
```

Comment:
okay! Will make proper changes. Was using string because I thought in some use cases this could be passed in as a string and Go might complain, but agree using `Int` is much cleaner.